src/LibraryManagementSystemApplication.java

```diff
@@ -8,15 +12,45 @@ public class LibraryManagementSystemApplication {
     public static void main(String[] args) {
 
     }
-    static void addBook(String title, String author, String ISBN, String pageNumber){
 
-         books[quantity][0] = title;
-         books[quantity][1] = author;
-         books[quantity][2] = ISBN;
-         books[quantity][3] = pageNumber;
+    static void addBook(String title, String author, String ISBN, String pageNumber) {
+
+        books[quantity][0] = title;
+        books[quantity][1] = author;
+        books[quantity][2] = ISBN;
+        books[quantity][3] = pageNumber;
 
-         quantity++;
+        quantity++;
 
         System.out.println("Kitabı Başarıyla Eklediniz!");
     }
-}
\ No newline at end of file
+
+    static void extendBooksArrayOnAddition(String title){
+        String[] newBooks = new String[books.length + 1];
+
+        for (int i = 0; i < books.length; i++){
+            newBooks[i] = Arrays.toString(books[i]);
```

Comment:
newBooks[i][0] = books[i][0]; 
newBooks[i][1] = books[i][1]; 
newBooks[i][2] = books[i][2]; 
newBooks[i][3] = books[i][3];

Reply:
updated it accordingly